### PR TITLE
New docs

### PR DESCRIPTION
**Pull request by Tyler-Keith-Thompson**
## Checklist:
- [x] Did you sign the [Contributor License Agreement](https://cla-assistant.io/wwt/SwiftCurrent)?
- [x] Is the linter reporting 0 errors?
- [x] Did you comply with our [styleguide](https://github.com/wwt/SwiftCurrent/blob/main/.github/STYLEGUIDE.md)?
- [x] Does the CI pipeline pass?
- [x] Did you [update the documentation](https://github.com/wwt/SwiftCurrent/blob/main/.github/CONTRIBUTING.md#documentation)?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,43 +42,45 @@ And just like that you're started!
...
+And just like that you've got a workflow! You can now add more items to it, or re-order the items that are there. To understand more of how this works and what is there [check out our developer docs](https://wwt.github.io/SwiftCurrent/How%20to%20use%20SwiftCurrent%20with%20SwiftUI.html)
 
-# Deep Dive
+# Check out our example apps
+We have [example apps](https://github.com/wwt/SwiftCurrent/tree/main/ExampleApps) for both SwiftUI and UIKit that show SwiftCurrent in action. They're even tested so you can see what it's like to test SwiftCurrent code. To run it locally, start by cloning the repo, open `SwiftCurrent.xcworkspace` and then run the `SwiftUIExample` scheme or the `UIKitExample` scheme. 
 
+# [See this to learn more](https://wwt.github.io/SwiftCurrent/index.html)
+For specific documentation check out:
 - [Why SwiftCurrent?](https://wwt.github.io/SwiftCurrent/why-this-library.html)
 - [Installation](https://wwt.github.io/SwiftCurrent/installation.html)
```
Comment (Richard-Gist) [inline README.md:80]: This installation file also needs to be updated because it says BETA.

**Review by morganzellers — APPROVED** [review]

File: .github/DocsPostProcessor/Package.swift [hunk 2]
```diff
@@ -0,0 +1,25 @@
+// swift-tools-version:5.5
```
Comment (Richard-Gist) [inline .github/DocsPostProcessor/Package.swift:1]: This is going to be too new for the pipeline.

You could update it to:
```suggestion
// swift-tools-version:5.4
```
and that should work.  Assuming Soup doesn't also require 5.5

File: .github/DocsPostProcessor/Sources/DocsPostProcessor/DocsPostProcessor.swift [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+            }
+        }
+        for file in files {
+            let html = try String(contentsOf: file, encoding: .utf8)
+            let doc: Document = try SwiftSoup.parse(html)
+            if replaceOverviewWithReadme {
+                try replaceOverviewWithReadme(document: doc)
+                print("Replaced overview with README for: \(file.absoluteString)")
+            }
+            if replaceReadmeVideoWithVimeoEmbed, file.lastPathComponent == "index.html" {
```
Comment (Richard-Gist) [inline .github/DocsPostProcessor/Sources/DocsPostProcessor/DocsPostProcessor.swift:37]: Not sure what this **should** be, but this causes a failure if you go to protocol docs and then try to go back to the Overview `SwiftCurrent/docs/Protocols/index.html`

File: README.md (hunk 1 above)
Comment (Tyler-Keith-Thompson) [reply README.md:80]: Should that wait for the PR that renames it? Or you wanna just do it here?

Comment (Richard-Gist) [reply README.md:80]: 🤷  I'm indifferent. As long as we don't forget.

Comment (Tyler-Keith-Thompson) [reply README.md:80]: We won't, it's on the list.

File: .github/abstract/Creating Workflows.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+
+### What is a `FlowRepresentable`?
+A `FlowRepresentable` is a protocol that describes a type that can appear in a `workflow`. It optionally declares `FlowRepresentable.WorkflowInput` and `FlowRepresentable.WorkflowOutput` indicating whether it requires data or whether it passes data forward. It also provides an initializer so that it can be created by the `workflow`. Once an item conforms to `FlowRepresentable` it is capable of being injected into any `workflow`. 
+
+### Valid workflow sequences
+Workflows enforce (either at compile-time or run-time) that the sequence of `FlowRepresentables` are well-formed. For example if a particular `FlowRepresentable` has a `FlowRepresentable.WorkflowInput` of `String` but the previous item passes an `Int` forward, that workflow is malformed. It's possible for a `FlowRepresentable` to declare a `FlowRepresentable.WorkflowInput` of `Never` if it does not take in data, `Any` if it can intake multiple types, or `AnyWorkflow.PassedArgs` if it can handle being passed no data, or data of any kind.
```
[the remark this answers is not in the document]
Comment (Tyler-Keith-Thompson) [reply .github/abstract/Creating Workflows.md]: Added

**Review by Richard-Gist — APPROVED** [review]
👍

**Pull request merged**